python/ray/train/tests/test_lightning_trainer.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+from ray.train.lightning import LightningConfigBuilder, LightningTrainer
+import ray
+from ray.air.util.data_batch_conversion import convert_batch_type_to_pandas
+import pytest
+
+from ray.train.tests.lightning_test_utils import (
+    LinearModule,
+    DoubleLinearModule,
+    DummyDataModule,
+)
+
+
+@pytest.fixture
+def ray_start_6_cpus_2_gpus():
+    address_info = ray.init(num_cpus=6, num_gpus=2)
+    yield address_info
+    # The code after the yield will run as teardown code.
+    ray.shutdown()
+
+
+def test_config_builder():
+    class DummyClass:
+        def __init__(self) -> None:
+            pass
+
+    with pytest.raises(ValueError):
+        LightningConfigBuilder().module(cls=DummyClass)
+
+    with pytest.raises(ValueError):
+        model = LinearModule(1, 1)
+        LightningConfigBuilder().module(cls=model)
+
+    with pytest.raises(TypeError):
+        LightningConfigBuilder().module(input_dim=10)
+
+    with pytest.raises(TypeError):
```

Comment:
can you add some simple ``match="..."`` to these pytest.raises() calls?
not just to make the tests better, also tells the reader what errors we are actually testing.

Reply:
Done